acquia_lift_profiles/qunit/tests.js

```diff
@@ -139,7 +139,16 @@ QUnit.asyncTest( "init test", function( assert ) {
 });
 
 
+QUnit.test("Get context values with 'do not track' cookie", function( assert ) {
+  $.cookie('tc_dnt', 'true', {path:'/'});
```

Comment:
It would be good to start the test showing that without the cookie being set, we get a non-empty object returned.